frontend/components/App.tsx

```diff
@@ -247,28 +232,24 @@ function App() {
                   <ConfigView
                     show={appViewState.showConfigDialog}
                     onClose={() =>
-                      appViewDispatch({ type: 'TOGGLE_CONFIG_DIALOG' })
+                      dispatch({ type: 'TOGGLE_CONFIG_DIALOG' })
                     }
                   />
                   <CreateDBDialog
                     show={appViewState.showCreateDialog}
                     DBInfo={DBInfo}
                     onClose={() =>
-                      appViewDispatch({ type: 'TOGGLE_CREATE_DIALOG' })
+                      dispatch({ type: 'TOGGLE_CREATE_DIALOG' })
```

Comment:
Replace current Context API code with correct redux function calls. You will need to import those functions from the slices